packages/application-components/src/components/internals/tabular-page.tsx

```diff
@@ -59,9 +62,28 @@ const CustomViewsSelectorWrapper = styled.div`
   margin: ${appKitDesignTokens.marginForCustomViewsSelectorAsTabular};
 `;
 
+const CustomViewsTabularPageContainer = (props: { children: ReactNode }) => {
+  const isRenderedInCustomView = Boolean(window.app.customViewId);
+  // wrapper which only appears when rendered in a Custom View
+  const Wrapper = isRenderedInCustomView ? 'div' : Fragment;
+  return (
+    <Wrapper
```

Comment:
Thanks for working on this fix.

I'm concerned about the solution though as we're overwriting children styles from a parent and also using the `!important` directive.

I haven't had the time to investigate an alternative so maybe we can merge this solution and I can give it a try as a follow-up.

Reply:
Can we define a new token that we can then override?

Reply:
Hmm but that's what the token is for, to allow overwriting.

Can you try to avoid using this wrapper container and define the token in the correct place?

Reply:
I'm not sure if I get this advice. Let me explain the current situation.
|  Page |  Header padding value needed in Custom/Internal Apps |  Header padding value needed Custom Views  | Comment |
|---|---|---|---|
| TabularMainPage |  `24px 40px 0` |  `0` |`paddingForTabularPageHeader` overridden in the custom views theme provider |
| TabularDetailPage  |  `24px 40px 0` | `0`  | `paddingForTabularPageHeader` overridden in the custom views theme provider |
| TabularModalPage  |  `24px 40px 0` | `24px 40px 0`  |`paddingForTabularModalPageHeader` token used from the main theme provider, nothing needs to be overridden in the custom views theme provider |

I'm not sure how to avoid using the wrapper and differentiating between TabularModalPage and TabularDetailPage/TabularMainPage tokens here: 
https://github.com/commercetools/merchant-center-application-kit/blob/d16ebd98c1b99a0c0c0e0f9a9df24cc8dcc0f542/packages/application-components/src/components/internals/tabular-page.tsx#L37-L46

Reply:
@emmenko As discussed, I applied the version referencing a styled component instead of `> div {`, all worked fine. 
Thanks for the suggestion 🙏